README.md

```diff
@@ -85,15 +85,22 @@ To list the available audio input devices:
 ./demo/c/build/cobra_demo_mic -s
 ```
 
+To list the available inference devices:
+
+```console
+./demo/c/build/cobra_demo_mic -i
+```
+
 To run the demo:
 
 ```console
-./demo/c/build/cobra_demo_mic -l ${LIBRARY_PATH} -a ${ACCESS_KEY} -d ${AUDIO_DEVICE_INDEX}
+./demo/c/build/cobra_demo_mic -l ${LIBRARY_PATH} -a ${ACCESS_KEY} -d ${AUDIO_DEVICE_INDEX} -y ${INFERENCE_DEVICE}
 ```
 
 Replace `${LIBRARY_PATH}` with path to appropriate library available under [lib](/lib), Replace `${ACCESS_KEY}` with
-AccessKey obtained from [Picovoice Console](https://console.picovoice.ai/), and `${INPUT_AUDIO_DEVICE}` with the index of
-your  microphone device.
+AccessKey obtained from [Picovoice Console](https://console.picovoice.ai/), `${INPUT_AUDIO_DEVICE}` with the index of
+your microphone device, and `${INFERENCE_DEVICE}` with the
+desired device to run Cobra on.
```

Comment:
we don't need these README changes, you can revert them - I was talking about the C SDK section in the main README.md which still looks like this:
```c
    pv_cobra_t *handle = NULL;
    pv_status_t status = pv_cobra_init(${ACCESS_KEY}, &handle);
    if (status != PV_STATUS_SUCCESS) {
        // error handling logic
    }
```
Device is required in C, so we need to add it. See a completed animals main README for what that looks like